ios/sound.mm

```diff
@@ -24,209 +24,279 @@
 #include "sound.h"
 #include <AVFoundation/AVFoundation.h>
 
+#define kOutputBus 0
+#define kInputBus  1
 
 /* Implementation *************************************************************/
-CSound::CSound ( void           (*fpNewProcessCallback) ( CVector<short>& psData, void* arg ),
+CSound::CSound ( void ( *fpNewProcessCallback ) ( CVector<short>& psData, void* arg ),
                  void*          arg,
                  const QString& strMIDISetup,
-                 const bool     ,
+                 const bool,
                  const QString& ) :
     CSoundBase ( "CoreAudio iOS", fpNewProcessCallback, arg, strMIDISetup ),
-    midiInPortRef ( static_cast<MIDIPortRef> ( NULL ) )
```

Comment:
So there's no midi support probably.

Reply:
iOS can do MIDI. Not me and my PR though :D

Reply:
;-).